.github/workflows/rust-compile.yml

```diff
@@ -121,3 +121,19 @@ jobs:
           ${{ steps.test-options.outputs.CARGO_TEST_OPTIONS}}
           --
           --nocapture
+      
+
+      # this will be removed after PR is approved
+      # it's very expensive to keep in the regular CI 
+      - name: Run pixi python test
+        if: contains(matrix.name, 'Windows')
+        shell: bash
+        run: |
+          cargo install pixi
+          mkdir pixi
+          cd pixi
+          pixi init
+          pixi add python
+          pwd
+          cd ../
+          cargo run -- rich --only-sdists -p /d/a/rip/rip/pixi/.pixi/env/python.exe
```

Comment:
So because we are not copying the `vcruntimeX.dll` and `pythonX.dll` over (as far as I can see?), I'm kinda unsure this test really works. It could be that these are accidentally on the path in the CI system. We could also try with a `python==3.6` i.e. an older version that might fail because the necessary dll's are not on the path.

Reply:
This I don't really get, if we do a `dumpbin /dependents` for an official python binary I think we still get the `vcruntime` and `pythonX` dlls. Also these are included in a conda python installation.

Reply:
When I download a stand-alone indiegreg python version I see these dll's:

<img width="338" alt="image" src="https://github.com/prefix-dev/rip/assets/417374/28d8c906-5d5f-49a6-b08a-996a21d959fb">

Reply:
Oh wait I think I get it now, there is a different "bigger" python in `venv/script/nt`. 

<img width="338" alt="image" src="https://github.com/prefix-dev/rip/assets/417374/ae8aeace-51f6-49c5-af44-f748c034158f">

Can't inspect it still but suspect there is more linked in there.